mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java

```diff
@@ -599,6 +600,26 @@ public void showUploadImageProgressbar(boolean b) {
         }
     }
 
+
+    public void loadImageGlide(int clientId) {
```

Comment:
- this method is called in all instances instead of async task.
- creates a glide url by adding url with the header as provided.
- provoked in two methods i.e.- showUploadImageFailed(String s) and showClientInformation(Client client)